src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,23 @@
-export const TodoList = () => {};
+import React from 'react';
+import { TodoInfo } from '../TodoInfo';
+
+type Todo = {
+  id: number;
+  title: string;
+  completed: boolean;
+  userId: number;
+};
+
+type TodoProps = {
+  todos: Todo[];
+};
+
+export const TodoList: React.FC<TodoProps> = ({ todos }) => {
+  return (
+    <section className="TodoInfo">
```

Comment:
```suggestion
    <section className="TodoList"">
```
compare with initial code and working example. It should be a TodoList class